### PR TITLE
Add possibility for set test Parameters when queued tests

### PR DESCRIPTION
**Pull request by Jedoku**

### CODE REVIEW

**Review by akihikodaki — CHANGES_REQUESTED** [review]
Use `hashtable` instead of inventing another syntax and parsing it.

File: toolsHCK.ps1 [hunk 1]
```diff
@@ -1404,7 +1404,17 @@ function gettestinfo {
...
+    param(
+        [Switch]$help, 
+        [String]$sup, 
+        [String]$IPv6, 
+        [Parameter(Position=1)][String]$test, 
+        [Parameter(Position=2)][String]$target, 
+        [Parameter(Position=3)][String]$project, 
+        [Parameter(Position=4)][String]$machine, 
+        [Parameter(Position=5)][String]$pool, 
+        [Parameter(Position=6)][String]$parameters
```
Comment (kostyanf14) [inline toolsHCK.ps1]: the `parameter` is not a mandatory option. In help should be `[-parameters <parameters>]`.

File: toolsHCK.ps1 [hunk 2]
```diff
@@ -1404,7 +1404,17 @@ function gettestinfo {
 # QueueTest
 function queuetest {
     [CmdletBinding()]
-    param([Switch]$help, [String]$sup, [String]$IPv6, [Parameter(Position=1)][String]$test, [Parameter(Position=2)][String]$target, [Parameter(Position=3)][String]$project, [Parameter(Position=4)][String]$machine, [Parameter(Position=5)][String]$pool)
+    param(
+        [Switch]$help,
```
Comment (kostyanf14) [inline toolsHCK.ps1]: Please split into 2 commit: refactoring of param list and adding new parameters

File: toolsHCK.ps1 [hunk 3]
```diff
@@ -1434,6 +1444,8 @@ function queuetest {
         Write-Output ""
         Write-Output "    poolname = The name of the pool."
         Write-Output ""
+        Write-Output "    parameters = Additional parameters in the format 'name1=value1;name2=value2'."
+        Write-Output ""
```
Comment (kostyanf14) [inline toolsHCK.ps1]: what if the value should be `my=password`?

**Review by kostyanf14 — CHANGES_REQUESTED** [review]

**Review by akihikodaki — APPROVED** [review]

**Review by kostyanf14 — APPROVED** [review]

**Pull request merged**